src/mobile/src/ui/views/onboarding/SeedReentry.js

```diff
@@ -99,15 +100,17 @@ class SeedReentry extends Component {
         if (isAndroid) {
             FlagSecure.deactivate();
         }
+        delete this.state.reenteredSeed;
     }
 
     /**
      * Navigates to set account name screen
      * @method onDonePress
      */
     onDonePress() {
-        const { t, seed, theme: { body } } = this.props;
-        if (this.state.seed === seed) {
+        const { t, theme: { body } } = this.props;
+        const { reenteredSeed } = this.state;
+        if (isEqual(reenteredSeed, global.onboardingSeed)) {
```

Comment:
Just thinking that maybe we should also have an extra check if `global.onboardingSeed` actually exists (as in not an empty Int8Array, undefined, null).

```
_.isEqual(undefined, undefined)
_.isEqual(null, null)
_.isEqual([], [])
``` 
They'll all return `true`.

Reply:
`global.onboardingSeed` cannot be null at this stage but I will add the check